SingularityService/src/main/java/com/hubspot/singularity/resources/RequestResource.java

```diff
@@ -149,7 +149,11 @@ public SingularityRequestParent submit(@ApiParam("The Singularity request to cre
 
     final long now = System.currentTimeMillis();
 
-    requestManager.activate(newRequest, maybeOldRequest.isPresent() ? RequestHistoryType.UPDATED : RequestHistoryType.CREATED, now, JavaUtils.getUserEmail(user));
+    if (maybeOldRequestWithState.isPresent() && maybeOldRequestWithState.get().getRequest().isLongRunning()) {
```

Comment:
There must have been a reason for it at some point. We actually have a test that ensures that they do get activated on request update